content/en/about/community/customers/index.md

```diff
@@ -61,6 +61,7 @@ Here are just a few of the organizations we know are using Istio. If you want to
     {{< company_logo link="https://www.skyscanner.com" logo="./skyscanner.svg" alt="Skyscanner" >}}
     {{< company_logo link="https://www.softonic.com" logo="./softonic.png" alt="Softonic S.L." >}}
     {{< company_logo link="https://www.spotify.com" logo="./spotify.svg" alt="Spotify" >}}
+    {{< company_logo link="https://www.splunk.com" logo="./splunk.png" alt="Splunk" >}}
```

Comment:
It seems that the order of you company is wrong.

Reply:
Ah good catch, thanks!